packages/ag-charts-community/src/chart/series/seriesAreaManager.ts

```diff
@@ -213,6 +229,7 @@ export class SeriesAreaManager extends BaseManager {
     private onHoverOrDrag(event: RegionEvent<'hover' | 'drag'>): void {
         this.highlight.pendingHoverEvent = event;
         this.hoverScheduler.schedule();
+        // this.hoverDevice = 'mouse';
```

Comment:
Nit: Needed?

Reply:
Well spotted. No it's not needed. I've removed this.